_pages/supermarkets.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Grocery shopping in Copenhagen
+layout: single
+permalink: register

```

Comment:
so this one needs to be something with like `topics/supermarkets`. and then it could be linked under [topics](https://cphbiosciencephd.github.io/info/topics)

Reply:
Thanks for the help! I changed it to topics/supermarkets.